setup.cfg

```diff
@@ -38,6 +38,7 @@ install_requires =
     requests >= 2.0
     setuptools
     tornado >= 4.4
+    importlib_metadata
```

Comment:
Should we say >= 4.6 to ensure we are getting the version compatible with Python 3.10 stdlib